gamemode/core/hooks/cl_hooks.lua

```diff
@@ -537,3 +537,20 @@ function GM:ScreenResolutionChanged(oldW, oldH)
 		nut.config.get("genericFont")
 	)
 end
+
+function GM:NutScriptLoaded()
+	for _, PLUGIN in pairs(nut.plugin.list) do
+		local author64ID = tonumber(PLUGIN.author)
+		local authorSteamID = string.find(PLUGIN.author, "STEAM")
```

Comment:
```suggestion
		local authorSteamID = string.match(PLUGIN.author, "STEAM_0:", 1) and PLUGIN.author or false
```
you don't want authorSteamID to be the starting pos of the string, you just want to see if it starts like a steamID

Reply:
That is true but I wouldn't recommend checking for "0:", also there's no need to define as it's simply being used as boolean

Reply:
Check for STEAM_ to avoid the rare but possible issue of someone starting their name with STEAM from causing problemos

Reply:
I highly doubt a plugin author will have a name starting with "STEAM_"